Source/WebCore/accessibility/AccessibilityRenderObject.cpp

```diff
@@ -1357,7 +1352,7 @@ bool AccessibilityRenderObject::computeAccessibilityIsIgnored() const
 
 int AccessibilityRenderObject::layoutCount() const
 {
-    if (!m_renderer || !is<RenderView>(*m_renderer))
+    if (!is<RenderView>(m_renderer))
         return 0;
     return downcast<RenderView>(*m_renderer).frameView().layoutContext().layoutCount();
```

Comment:
The compiler would not let me convert this one. Something didn't line up.

Reply:
It was similar to what GCC apparently outputs:
```
/app/webkit/Source/WebCore/accessibility/AccessibilityRenderObject.cpp:2486:70: note:   mismatched types ‘auto*’ and ‘WTF::WeakPtr<WebCore::RenderListItem, WTF::SingleThreadWeakPtrImpl, WTF::RawPtrTraits<WTF::SingleThreadWeakPtrImpl> >’
 2486 |     auto* renderListItem = dynamicDowncast<RenderListItem>(m_renderer);
```
@cdumez ideas?

Reply:
`auto* renderListItem = dynamicDowncast<RenderListItem>(m_renderer.get());` would work. `m_renderer` is a WeakPtr and `dynamicDowncast<>()` doesn't know how to handle WeakPtrs (or CheckedPtrs) at the moment. Only raw pointers and Ref/RefPtr are currently supported.